cmd/olympus/actions/storage.go

```diff
@@ -34,12 +33,6 @@ func GetStorage() (storage.Backend, error) {
 			return nil, err
 		}
 		return mongo.NewStorage(mongoURI)
-	case "postgres", "sqlite", "cockroach", "mysql":
```

Comment:
we are removing postgres support, is that right? we keep postgres image running and we have olympus database yaml saying `dialect: "postgres"`

Reply:
We need to have Postgres image and the yml file on Olympus because we have a CDN RDBMS driver.  This way, we can run tests for CDN but still remove RDBS storage support from pkg/